root/static/scripts/relationship-editor/common/dialog.js

```diff
@@ -375,6 +375,10 @@ class Dialog {
     relationship.entities(relationship.entities().slice(0).reverse());
   }
 
+  toggleDatesHelp() {
+    this.showDatesHelp(!this.showDatesHelp.peek());
```

Comment:
I'm assuming you don't need to call preventDefault here, since `toggleAttributesHelp` doesn't appear to (since with `href="#"` the scroll position would normally jump to the top of the page).